packages/appeals-service-api/src/routes/v2/appeals/repo.js

```diff
@@ -5,6 +5,52 @@ const { PrismaClientKnownRequestError } = require('@prisma/client/runtime/librar
  * @typedef {import("@prisma/client").Prisma.AppealUserGetPayload<{include: {Appeals: {include: {Appeal: { include: {AppealCase: true }}}}}}>} UserWithAppeals
  */
 
+// const caseDashboardSelect = {
```

Comment:
did you mean to leave this comment here?

Reply:
eh I was hoping to add this in at some point, will remove for now